pulseaudio-control.bash

```diff
@@ -4,12 +4,13 @@
 # Polybar Pulseaudio Control                                     #
 # https://github.com/marioortizmanero/polybar-pulseaudio-control #
 ##################################################################
-
```

Comment:
This line removal isn't really necessary, I think it's nice to separate the header from the code?